doc/source/development/meeting.rst

```diff
@@ -0,0 +1,32 @@
+.. _meeting:
+
+==================
+Developer Meetings
+==================
+
+We hold regular developer meetings on the second Wednesday
+of each month at 12:00 US Central Time. These meetings and their minutes are open to
```

Comment:
Can you add U
```suggestion
of each month at  18:00 UTC / 12:00 US Central Time. These meetings and their minutes are open to
```

Reply:
Hmm, I suppose the 12:00 central time is robust to summer/winter, and the UTC not?

Reply:
Indeed it's not robust :) Right now the calendar invite is set to US Central time (since I created it 😄).

But which do we want?

1. A specific US timezone (So it's always the same local time for most US people. Will shift once or twice for people on other DST calendars).
2. UTC (So the local time will shift for people doing DST transitions).

cc @pandas-dev/pandas-core.

How about a 🎉 if you want option 1 and 🚀 if you want option 2?

Reply:
I am fine with the 12:00 central time (so using local time), I mainly wanted to have this translated to a local time that is familiar for a European as well :)

DST switches are not exactly equal (US: 2nd sunday of March, Europe: last sunda of March), but so since we are the second Wednesday, it's only in March that there can be a difference, and not even every year I think :)

So maybe you can put "19:00 Europe/Brussels" instead of the UTC ?

Reply:
I'm biased, because I'm in UTC, but I have a small preference for it. I feel people outside the US may somehow feel this is not for them, and I think we want the opposite. I think UTC feels neutral. If most people understand US Central time better than other time zones, may be we can just leave it, and add a small note (e.g. people from anywhere in the world welcome).

What would be good if to have the links to the calendars somewhere, so people can subscribe, and see in their current time zone (that's the main way I manage to know when are events).

Reply:
Seems like the UTCs have it.

> What would be good if to have the links to the calendars somewhere, so people can subscribe, and see in their current time zone (that's the main way I manage to know when are events).

This PR includes links, which you can subscribe to / import into your calendar. Or do you mean something else?